libs/designsystem/src/lib/components/list/directives/infinite-scroll.directive.ts

```diff
@@ -54,8 +61,30 @@ export class InfiniteScrollDirective implements AfterViewInit, OnDestroy {
 
   constructor(private elementRef: ElementRef, private window: WindowRef, private zone: NgZone) {}
 
-  ngAfterViewInit(): void {
+  ngAfterViewInit() {
+    this.checkScroll$Subscription(this.disabled);
+  }
+
+  /**
+   * On element scroll event emit next {@link scroll$} observable value
+   */
+  @HostListener('window:scroll')
+  onScroll(): void {
     if (this.disabled) return;
+    const scroll = this.getScroll();
+    this.scroll$.next(scroll);
+  }
+
+  /**
+   * trigger {@link ngUnsubscribe} complete on component destroy lifecycle hook
+   */
+  ngOnDestroy(): void {
+    this.ngUnsubscribe$.next();
+    this.ngUnsubscribe$.complete();
+  }
+
+  private checkScroll$Subscription(disabled: boolean) {
```

Comment:
The `$` suffix is a naming convention to denote observable variables - we don't use it inside member names

Reply:
Should be resolved by 6730130